libflux/flux/src/lib.rs

```diff
@@ -998,8 +977,8 @@ from(bucket: v.bucket)
         let ast = Box::into_raw(Box::new(ast.into()));
         let errh = unsafe { flux_ast_get_error(ast) };
         assert_eq!(
-            "error at test@1:9-1:10: invalid expression: invalid token for primary expression: DIV",
-            format!("{}", errh.unwrap().err)
+            CString::new("error at test@1:9-1:10: invalid expression: invalid token for primary expression: DIV").unwrap(),
```

Comment:
Can you use `into_string()` on the line below instead of `CString` here? I'd prefer to keep the Rust test harness using either `String` or `&str`.